Core/src/main/java/com/github/intellectualsites/plotsquared/plot/object/worlds/SinglePlotArea.java

```diff
@@ -54,8 +55,8 @@ public void loadWorld(final PlotId id) {
         SetupObject setup = new SetupObject();
         setup.plotManager = "PlotSquared:single";
         setup.setupGenerator = "PlotSquared:single";
-        setup.type = TYPE;
-        setup.terrain = TERRAIN;
+        setup.type = getType();
+        setup.terrain = getTerrain();
```

Comment:
Feels like we should update everything, so we don't mix directly using variables with encapsulated variables.

Reply:
You mean the `SetupObject`? I agree, altough this isn't really part of the API, so we can change it at any time as we want.